ext/openssl/lib/openssl/ssl.rb

```diff
@@ -21,7 +21,7 @@ module OpenSSL
   module SSL
     class SSLContext
       DEFAULT_PARAMS = {
-        :ssl_version => "SSLv23",
+        :ssl_version => "TLSv1_2",
```

Comment:
I'm wondering as well (not a ruby expert). This code does not define `SSL_OP_NO_TLSv1_1` at all, so maybe it will remain enabled by default?